--- a/SConstruct	2015-05-10 01:34:58.000000000 +0200
+++ b/SConstruct	2015-05-23 14:33:51.186635813 +0200
@@ -67,11 +67,6 @@
     context.Message('Checking for git revision... ')
     rev = STATIC_GIT_REV
 
-    try:
-        rev = subprocess.check_output('git log --pretty=format:"%h" -n 1', shell=True)
-    except subprocess.CalledProcessError:
-        print('Unable to find git revision.')
-
     rev = rev or 'unknown'
     conf.env['gitrev'] = rev
     context.Result(rev)
