--- Makefile~	2015-06-27 15:28:09.743884616 +0200
+++ Makefile	2015-06-27 15:28:39.670911682 +0200
@@ -100,11 +100,11 @@
 GREP           ?= grep
 PYTHON         ?= python
 UNIX2DOS       ?= unix2dos
-UNIX2DOS_FLAGS ?= $(shell [ -n $(UNIX2DOS) ] && $(UNIX2DOS) -q --version 2>/dev/null && echo "-q" || echo "")
+UNIX2DOS_FLAGS ?= $(shell [ -n $(UNIX2DOS) ] && $(UNIX2DOS) -q --version 2>/dev/null >&2 && echo "-q" || echo "")
 
 # Graphics processing
-GIMP           ?= gimp
-GIMP_FLAGS     ?= -n -i
+#GIMP           ?= gimp
+#GIMP_FLAGS     ?= -n -i
 
 # NML
 NML            ?= nmlc
@@ -714,7 +714,7 @@
 	$(_E)
 endif
 	$(_E) "Release:"
-	$(_E) "bananas:     Upload bundle to BaNaNaS
+	$(_E) "bananas:     Upload bundle to BaNaNaS"
 	$(_E)
 	$(_E) "Valid command line variables are:"
 	$(_E) "Helper programmes:"
